inspirehep/modules/converttohal/tei.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
```

Comment:
The [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) is only useful if you expect users to run this script directly, i.e. `$ ./tei.py`. If you expect users to import it and use the functions inside, it doesn't matter.